### PR TITLE
Nina/katsu dataset

### PR DESCRIPTION
**Pull request by ninawang-uhn**
Hi Jonathan, I added sample minimum required field json, create test dataset python script and pytests for the stack. I figured it's already too giant for a pull request, so I think maybe you can review it. Sorry for such a giant PR. Securing biosamples endpoint is not working right now. I am figuring it out currently.

### CODE REVIEW

File: tests/create_katsu_datasets.sh [hunk 1]
```diff
@@ -3,32 +3,32 @@ cd tests
 export PROJECT_ID=$(curl -X POST 'localhost:8001/api/projects' -H "Content-Type: application/json" -d '{"title": "tests", "description": "Project for tests"}'|jq -r .identifier)
 echo "Created project ${PROJECT_ID}";echo
 
-export OPEN_DATASET1_ID=$(cat dataset_template.json|sed s/DATASET_TITLE/open1/|sed s/PROJECT_ID/${PROJECT_ID}/|curl -X POST 'localhost:8001/api/datasets' \
+export OPEN_DATASET1_ID=$(cat templates/dataset_template.json|sed s/DATASET_TITLE/open1/|sed s/PROJECT_ID/${PROJECT_ID}/|curl -X POST 'localhost:8001/api/datasets' \
 -H 'Content-Type: application/json' \
```
Comment (ljdursi) [inline tests/create_katsu_datasets.sh:7]: I'd suggest pulling out this repetitive cat/sed/sed/curl one-liner into a bash function

File: tests/create_katsu_test_datasets.py [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+    response = requests.post(url, json=data)
+    print(response.json())
+    return response.json()
+
+
+'''
+does a post request to /api/projects
+creates a project with the project title passed in
+returns the project id
+'''
```
Comment (ljdursi) [inline tests/create_katsu_test_datasets.py:18]: Looks good - let's put these comments as docstrings within the function:

```
def get_project_id(project_title):
    '''
    does a post request....
    '''
```

That way if someone does e.g. `help(get_project_id)` they'll get the doctoring

File: docker-compose.yml [hunk 3]
```diff
@@ -129,6 +127,7 @@ services:
       POSTGRES_USER: admin
       POSTGRES_PASSWORD: admin
       CANDIG_OPA_URL: https://opa:8181
+      CANDIG_OPA_VERSION: rego_dev_playground
```
Comment (ljdursi) [inline docker-compose.yml:130]: I take it that this is to distinguish between the format that's being used here and what's currently being used in candig/katsu?  Let's see if we can come up with a better naming scheme than this

File: tests/create_katsu_datasets.sh [hunk 4]
```diff
@@ -43,17 +43,17 @@ echo "Created individual with id test_individual";echo
 counter=1
 for i in $OPEN_DATASET1_ID $OPEN_DATASET2_ID $REGISTERED_DATASET3_ID $CONTROLLED_DATASET4_ID $CONTROLLED_DATASET5_ID $CONTROLLED_DATASET6_ID
 do
-    cat table_ownership_template.json|sed s/TABLE_ID/${counter}/|sed s/DATASET_ID/${i}/|curl -X POST 'localhost:8001/api/table_ownership' \
+    cat templates/table_ownership_template.json|sed s/TABLE_ID/${counter}/|sed s/DATASET_ID/${i}/|curl -X POST 'localhost:8001/api/table_ownership' \
```
Comment (ljdursi) [inline tests/create_katsu_datasets.sh:46]: Same here with refactoring this into a bash function

**Review by ljdursi — CHANGES_REQUESTED** [review]
Looks good!  I like how comprehensive the tests are.  I've requested some minor changes.

File: docker-compose.yml (hunk 3 above)
Comment (ninawang-uhn) [reply docker-compose.yml:130]: Yes, it is used in katsu to distinguish between what's used in dycons and ours.

**Pull request merged**